httpbp/admin.go

```diff
@@ -15,5 +18,9 @@ import (
 // This function blocks, so it should be run as its own goroutine.
 func ServeAdmin(healthCheck HandlerFunc) {
 	admin.Mux.Handle("/health", handler{handle: healthCheck})
-	log.Warnw("httpbp: admin serving exited", "err", admin.Serve())
+	if err := admin.Serve(); errors.Is(err, http.ErrServerClosed) {
+		log.Info("httpbp: server closed")
+	} else {
+		log.Panicw("httpbp: admin serving exited", "err", err)
```

Comment:
maybe you want a word stronger than "exited"? maybe "failed"?

Reply:
I kept the language from the initial log message. Agreed that a stronger message is better. I'll update